### ISSUE TITLE
Don't you have to update batch norm statistics?

### ISSUE THREAD

**Issue by sh1ng** [ikhlestov/vision_networks#3]
Hi,

thank you for the implementation!
I can't find update stat operation
from the documentation 
`update_ops = tf.get_collection(tf.GraphKeys.UPDATE_OPS) with tf.control_dependencies(update_ops): train_op = optimizer.minimize(loss)`

https://www.tensorflow.org/api_docs/python/tf/contrib/layers/batch_norm

**Comment by ikhlestov** [comment]
Hi,

I've provide additional argument `updates_collections=None` in call to the batch norm([link](https://github.com/ikhlestov/vision_networks/blob/master/models/dense_net.py#L294)) to force the updates in place as per another approach from docs.
